packages/common/src/ui/components/portals/AppBarButtons/AppBarButtons.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+import { act, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { AppBarButtons, AppBarButtonsPortal } from './AppBarButtons';
+
+describe('AppBarContent', () => {
```

Comment:
`AppBarButtons` ...of course, I'd never have the wrong name on a test suite 🙄

Reply:
🙈 

Hahha!

I'm being lazy and fixing this down the line